planning/mv-params.json

```diff
@@ -0,0 +1,151 @@
+the user submits something that has this format:
+
+{
+    "includes": [
```

Comment:
"common" becomes "includes"

Reply:
glory to God! +1

Reply:
I'm wondering if I have forgotten a reason why we didn't do this initially.  It seems so obvious, there must be a reason...

Reply:
I am not exactly thrilled with include and includes, but I can't think of anything else, so it's fine by me

Reply:
so this is the time to explore alternatives if there is something you don't like about it.  what is it that you don't like about includes/include?

Reply:
I don't like the fact that they are so similar, and I am worried that someone will accidentally get them swapped.  Perhaps it's just that a verb is being treated as a noun (includes), and it makes my brain twitch a bit.  I think I am fine with "include", but perhaps "includes" could be something else, like "presets" or "stubs".

Reply:
That is a valid concern.  That said, the schema validation should make this a trivial issue that gets flagged immediately and prevents downstream catastrophe.

It is worth noting that we are already using "vals" and "val" -- albeit "vals" is in mv-params and "val" is in bench-params.  So a similar concern is present there.

Reply:
I don't mind the plural aspect at all; it's the verb/noun thing.  Vals vs val, that does not bother me because they are in two different files, and we really want to enforce that in mval-params, it is capable of being plural.

@mrbojangles3 any preferences here?

Reply:
As a new user to the workflow, common seemed to make sense to me, as it would be common to every permutation. of the config file. If we don't want "includes" maybe try "common_options" or "global_options" ?

Reply:
> As a new user to the workflow, common seemed to make sense to me, as it would be common to every permutation of the config file. If we don't want "includes" maybe try "common_options" or "global_options" ?

+1 for **global_options** in the common block, and **include** for importing it.

Reply:
That seems fine to me.  Should it be "global_options" or "global-options"?  Since this isn't bash we don't have to avoid hyphens if we don't want to...

Reply:
"-"